### PR TITLE
Image: Set up the `Debian_64` machine

### PR DESCRIPTION
**Pull request by Hywan**
Address https://github.com/hoaproject/ActionBoard/issues/11.

This is WIP.

### CODE REVIEW

File: MachineImages/Templates/Debian_i386/Http/Preseed.cfg [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+
+# Uncomment this to add multiarch configuration for i386
+#d-i apt-setup/multiarch string i386
+
+
+### Package selection
+tasksel tasksel/first multiselect standard
+
+# Individual additional packages to install
+d-i pkgsel/include string openssh-server build-essential
```
Comment (jubianchi) [inline MachineImages/Templates/Debian_i386/Http/Preseed.cfg]: id this is a production image I don't think build-essential is required

File: MachineImages/Templates/Debian_i386/Makefile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+all: up
```
Comment (jubianchi) [inline MachineImages/Templates/Debian_i386/Makefile]: there is no up target in this Makefile

File: MachineImages/Templates/Debian_i386/Http/Preseed.cfg (hunk 1 above)
Comment (Hywan) [inline MachineImages/Templates/Debian_i386/Http/Preseed.cfg]: For the provisionning, I think this is required :-/.

File: MachineImages/Templates/Debian_i386/Template.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    ],
+    "provisioners": [
+        {
+            "type"   : "shell",
+            "scripts": [
+                "Scripts/Initialize.sh"
+            ]
+        }
+    ]
+}
```
Comment (jubianchi) [inline MachineImages/Templates/Debian_i386/Template.json]: missing newline at end of file

Comment (Hywan) [inline MachineImages/Templates/Debian_i386/Template.json]: Thanks!

File: MachineImages/Templates/Debian_i386/Makefile [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+build:
+	packer build Template.json
+	mv -f -v output-debian-i386-vbox/ ../../Artifacts/Debian_i386/
+
+debug-build:
+	PACKER_LOG=1 PACKER_LOG_PATH=Packer.log packer build -debug  Template.json
```
Comment (jubianchi) [inline MachineImages/Templates/Debian_i386/Makefile]: debug flag will prevent automated build as it usually requires user inputs

File: MachineImages/Templates/Debian_i386/Template.json [hunk 5]
```diff
@@ -0,0 +1,39 @@
+{
+    "variables": {
+        "name"    : "debian-i386",
+        "headless": "false"
+    },
+    "builders": [
+        {
+            "type"             : "virtualbox-iso",
```
Comment (jubianchi) [inline MachineImages/Templates/Debian_i386/Template.json]: for production I would rather build for vmware or qemu

File: MachineImages/Templates/Debian_i386/Makefile (hunk 2 above)
Comment (Hywan) [inline MachineImages/Templates/Debian_i386/Makefile]: Correct. I am making something more coherent.

File: MachineImages/Templates/Debian_i386/Makefile (hunk 4 above)
Comment (Hywan) [inline MachineImages/Templates/Debian_i386/Makefile]: Yup, but this is for the `debug-build`, not the automated `build` target.

File: MachineImages/Templates/Debian_i386/Template.json (hunk 5 above)
Comment (Hywan) [inline MachineImages/Templates/Debian_i386/Template.json]: Yup, I am going to export it as a variable.

Comment (Hywan) [inline MachineImages/Templates/Debian_i386/Template.json]: And we are going to target AWS :-).

File: MachineImages/Templates/Debian_i386/Template.json (hunk 3 above)
Comment (Hywan) [inline MachineImages/Templates/Debian_i386/Template.json]: Done.

File: MachineImages/Templates/Debian_i386/Makefile [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+
+build:
+	packer build Template.json
+	mv -f -v output-debian-i386-vbox/ ../../Artifacts/Debian_i386/
+
+debug-build:
+	PACKER_LOG=1 PACKER_LOG_PATH=Packer.log packer build -debug Template.json
+
+clean:
+	rm -r packer_cache
```
Comment (jubianchi) [inline MachineImages/Templates/Debian_i386/Makefile]: adding `-f` to both `rm` commands will prevent failure if files/directories do not exist

File: MachineImages/Templates/Debian_i386/Http/Preseed.cfg (hunk 1 above)
Comment (jubianchi) [inline MachineImages/Templates/Debian_i386/Http/Preseed.cfg]: you are not building anything in the preseed nor in the init script so I would say that this package is useless ;)

Comment (Hywan) [inline MachineImages/Templates/Debian_i386/Http/Preseed.cfg]: I agree. I am going to test.

However, see this issue: https://github.com/hoaproject/Infrastructure/issues/2.

File: MachineImages/Templates/Debian_i386/Template.json [hunk 7]
```diff
@@ -0,0 +1,41 @@
+{
+    "variables": {
+        "name"             : "debian-i386",
```
Comment (Pierozi) [inline MachineImages/Templates/Debian_i386/Template.json]: Must better to take 64bit version `debian-amd64` `http://cdimage.debian.org/debian-cd/8.2.0/amd64/iso-cd/debian-8.2.0-amd64-netinst.iso`

Comment (Hywan) [inline MachineImages/Templates/Debian_i386/Template.json]: Done.

**Comment by Hywan** [comment]
I am currently testing the 64bits version.

**Pull request merged**